resources/assets/js/pages/page-show.js

```diff
@@ -161,6 +161,8 @@ let setupPageShow = window.setupPageShow = function (pageId) {
         }
     });
 
+    // in order to call from other places.
+    window.setupPageShow.goToText = goToText;
```

Comment:
This has been done because I need to use it to highlight directly linked comments.